Makefile

```diff
@@ -15,5 +15,7 @@
 CMDS=csi-provisioner
 all: build
 
+# TODO: Change in csi-release-tools before PR merge?
+export CSI_PROW_GO_VERSION_BUILD=1.21
```

Comment:
Yes, please send a PR for csi-release-tools. All sidecar releases for 1.29 will need a new go.

Reply:
Working on a PR, for now I modified the directory directly to hopefully unblock CI, although this will cause `make test` to fail because it detects non-upstreamed changes